api/Text.json

```diff
@@ -0,0 +1,341 @@
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": true
+            },
+            "safari_ios": {
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "assignedSlot": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Slotable/assignedSlot",
+          "support": {
+            "chrome": {
+              "version_added": "53"
+            },
+            "chrome_android": {
+              "version_added": "53"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": null
+            },
+            "opera": {
+              "version_added": "40"
+            },
+            "opera_android": {
+              "version_added": "40"
+            },
+            "safari": {
+              "version_added": "10.1"
+            },
+            "safari_ios": {
+              "version_added": "10.1"
+            },
+            "samsunginternet_android": {
+              "version_added": null
+            },
+            "webview_android": {
+              "version_added": "53"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "replaceWholeText": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Text/replaceWholeText",
+          "support": {
+            "chrome": {
+              "version_added": true,
+              "version_removed": "45"
+            },
+            "chrome_android": {
+              "version_added": true,
+              "version_removed": "45"
+            },
+            "firefox": {
+              "version_added": true,
+              "version_removed": "10"
+            },
+            "firefox_android": {
+              "version_added": true,
+              "version_removed": "10"
+            },
+            "ie": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": true,
+              "version_removed": "32"
+            },
+            "opera_android": {
+              "version_added": true,
+              "version_removed": "32"
+            },
+            "safari": {
+              "version_added": true
```

Comment:
I've tested this in Safari, and it is no longer supported. The same is probably true for iOS. Should we just put false for now? That would be less misleading than saying it is true.